pkgs/servers/computing/slurm/default.nix

```diff
@@ -25,8 +30,8 @@ stdenv.mkDerivation rec {
     ] ++ stdenv.lib.optional (gtk2 == null)  "--disable-gtktest";
 
   preConfigure = ''
-    substituteInPlace ./doc/html/shtml2html.py --replace "/usr/bin/env python" "${python.interpreter}"
-    substituteInPlace ./doc/man/man2html.py --replace "/usr/bin/env python" "${python.interpreter}"
+    patchShebangs ./doc/html/shtml2html.py
+    patchShebangs ./doc/man/man2html.py
```

Comment:
Solution is ok. Often I just prefer the shotgun method and use `patchShebangs .` to avoid future shebangs maintenance.